Source/WebCore/page/LocalFrameViewLayoutContext.h

```diff
@@ -192,6 +197,7 @@ class LocalFrameViewLayoutContext {
     std::unique_ptr<Layout::LayoutTree> m_layoutTree;
     std::unique_ptr<Layout::LayoutState> m_layoutState;
     std::unique_ptr<UpdateScrollInfoAfterLayoutTransaction> m_updateScrollInfoAfterLayoutTransaction;
+    HashMap<const RenderLayerModelObject*, Vector<WeakPtr<RenderLayerModelObject>>> m_descendantsNeedingTransformUpdate;
```

Comment:
`WeakHashMap` ?

Reply:
The "descendants" in the name is confusing; here s a member variable. Would be better as `m_containersWith...`

Reply:
Fixed.